### PR TITLE
LPS-156380 Hide default icon and show clay icon instead

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/java/com/liferay/fragment/collection/contributor/inputs/InputsFragmentCollectionContributor.java [hunk 1]
```diff
@@ -26,7 +26,7 @@
  * @author Pablo Molina
  */
 @Component(
-	enabled = false, property = "fragment.collection.key=INPUTS",
+	enabled = true, property = "fragment.collection.key=INPUTS",
```
Comment (ealonso) [inline modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/java/com/liferay/fragment/collection/contributor/inputs/InputsFragmentCollectionContributor.java:29]: @victorg1991 I think you should keep it as false

Comment (victorg1991) [reply modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/java/com/liferay/fragment/collection/contributor/inputs/InputsFragmentCollectionContributor.java:29]: No!

Comment (victorg1991) [reply modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/java/com/liferay/fragment/collection/contributor/inputs/InputsFragmentCollectionContributor.java:29]: I want to enable this feature 😤

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
@victorg1991 se should review it tomorrow

**Pull request closed without merging**